### DISCUSSION TITLE
Hide sponsor button if it would be sponsoring me

### DISCUSSION THREAD

**Discussion by pxeger** [community/community#6800] [Other Feature Feedback, Questions, & Ideas]
At the bottom of some repository pages, a button is shown which says

> Show your support for [...] by sponsoring them. 

However, this is shown on repositories which belong to organisations which I own, and which don't even have sponsorship set up. This surely should not be the case (for two reasons: 1: if I own the organisation, it doesn't make sense for me to sponsor myself; 2: the organisation doesn't have sponsorship set up so it can't be sponsored)

Also, if I click the sponsor button, because sponsorship isn't set up, it takes me to the "join waitlist" page (https://github.com/sponsors/.../waitlist)

Here is a screenshot of an example in https://github.com/attempt-this-online/attempt-this-online
![screenshot of issue page in attempt-this-online repository with sponsor button at the bottom of the page, highlighted](https://user-images.githubusercontent.com/67427173/139027240-b422789d-97d9-47f0-b429-a1064e948404.png)

**Comment by cheshire137** [comment]
Hi there, thanks for the report! I'm one of the devs on the Sponsors team. We did intentionally choose to show the button to maintainers themselves, based on feedback from some about wanting to see what others in their community see, but the button appearing when the org is not able to be sponsored at all looks like a bug! We'll look into this, thanks!
